lang/perl-net-ssleay/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=perl-net-ssleay
+PKG_VERSION:=1.94
+PKG_RELEASE:=1
+
+PKG_SOURCE_NAME:=Net-SSLeay
+PKG_SOURCE_URL:=https://cpan.metacpan.org/authors/id/C/CH/CHRISN
+PKG_SOURCE:=$(PKG_SOURCE_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=9d7be8a56d1bedda05c425306cc504ba134307e0c09bda4a788c98744ebcd95d
+PKG_BUILD_DIR:=$(BUILD_DIR)/perl/$(PKG_SOURCE_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Jens Wagner <jens@wagner2013.de>
+PKG_LICENSE:=GPL-1.0-or-later Artistic-1.0-Perl
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=openssl
```

Comment:
Same question here. In depands, there is +libopenssl. Do you need to explicitly specify it here?